### PR TITLE
Update Readme. Stop using Git Tags (doesn't work with squash/merge).

### PR DESCRIPTION
**Pull request by ChristianJHughes**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,23 +4,52 @@ Ramsey Solutions Frontend configurations and standards.
...
+### New Packages
+
+**To create a brand new package:**
+
+1. Run the `lerna create` command to create a new Lerna managed package.
+2. Ensure that the `private` property is set to `true` before the package is ready for publication.
+
+Once ready for publication to NPM:
+
+1. Open a feature branch for your release
```
Comment (alexmacarthur) [inline README.md:38]:
```suggestion
1. Open a feature branch for your release.
```

File: README.md [hunk 2]
```diff
@@ -4,23 +4,52 @@ Ramsey Solutions Frontend configurations and standards.
 
 ## Packages
 
+Packages are published to NPM within the [`ramseyinhouse`](https://www.npmjs.com/org/ramseyinhouse) organization.
```
Comment (alexmacarthur) [inline README.md:7]: I think the appropriate way to refer to it is `npm`, even in written copy. It's this way in their [own copy](https://www.npmjs.com/get-npm), as well as external sources like [Wikipedia](https://en.wikipedia.org/wiki/Npm_(software)).

```suggestion
Packages are published to npm within the [`ramseyinhouse`](https://www.npmjs.com/org/ramseyinhouse) organization.
```

File: README.md [hunk 3]
```diff
@@ -4,23 +4,52 @@ Ramsey Solutions Frontend configurations and standards.
...
+
+[![npm version](https://badge.fury.io/js/%40ramseyinhouse%2Fstylelint-config.svg)](https://badge.fury.io/js/%40ramseyinhouse%2Fstylelint-config)
+
+[`@ramseyinhouse/stylelint-config`](./packages/stylelint-config/README.md)
+
+## About
+
+This repository contains multiple packages managed through [Lerna](https://lerna.js.org/).
+
+Package versions are automatically published to [NPM](https://www.npmjs.com/org/ramseyinhouse) via [Github Actions](./.github/workflows/main.yml) as they are updated.
```
Comment (alexmacarthur) [inline README.md:25]:
```suggestion
Package versions are automatically published to [npm](https://www.npmjs.com/org/ramseyinhouse) via [Github Actions](./.github/workflows/main.yml) as they are updated.
```

File: README.md [hunk 4]
```diff
@@ -4,23 +4,52 @@ Ramsey Solutions Frontend configurations and standards.
...
 ## Contributing
 
+### New Packages
+
+**To create a brand new package:**
+
+1. Run the `lerna create` command to create a new Lerna managed package.
+2. Ensure that the `private` property is set to `true` before the package is ready for publication.
+
+Once ready for publication to NPM:
```
Comment (alexmacarthur) [inline README.md:36]:
```suggestion
Once ready for publication to npm:
```

File: README.md [hunk 5]
```diff
@@ -4,23 +4,52 @@ Ramsey Solutions Frontend configurations and standards.
...
 ```bash
-yarn && yarn run release
+yarn run release
 ```
 
-_**Note:** The `release` command will increase `package.json` version(s) based on the scope of recent changes. A commit will be made automatically._
+_The `release` command will allow you to interactively update the `package.json` version(s) based on the scope of recent changes. A commit will be created and pushed automatically._
 
-3. Push the feature branch, and open a pull request. Once merged, the new package version(s) will be published to NPM automatically via GitHub Actions.
+3. Open a pull request. Once merged, the new package version(s) will be published to NPM automatically.
```
Comment (alexmacarthur) [inline README.md:55]:
```suggestion
3. Open a pull request. Once merged, the new package version(s) will be published to npm automatically.
```

**Review by alexmacarthur — CHANGES_REQUESTED** [review]
Nice work! Everything reads great -- just a couple of small things.

**Comment by ChristianJHughes** [comment]
@alexmacarthur Thank you, updates made! Could you take another look?

**Review by alexmacarthur — APPROVED** [review]
🚀

**Pull request merged**